portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,159 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  background-color: rgb(29, 48, 61);
+  color: whitesmoke;
+  width: 60%;
+  margin: auto;
+  font-size:20px;
+}
+
+h2 {
+  font-size: 30px;
+}
+h3 {
+  text-align: center;
+}
+
+header {
+  display: flex;
+  margin: auto;
+  height: 92px;
+  align-items: center;
+}
+
+#heading {
+  flex: 1;
+}
+
+nav {
+  flex: 2;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+#intro {
+  display: flex;
+}
+
+#intro-par {
+  margin: 0 auto auto 2%;
 }
 
 #greeting-container {
+margin-top: 20px;
+}
+
+#prof-pic {
+  margin: auto;
+  height: 300px;
+  width: 300px;
+  border-radius: 50px;
+}
+
+.fa, .fab {
+  padding: 20px;
+  font-size: 45px;
+  text-align: center;
+}
+
+.fa:hover, .fab:hover {
+  color:rgb(165, 165, 165);
+}
+
+a, a:visited, a:hover, a:active {
+  text-decoration: none;
+  color: white;
+}
+
+footer {
+  position: absolute;
+  margin: auto;
+  margin-left: 25%;
+  bottom:20px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+li > a {
```

Comment:
You have intentionally made this section have no spaces between the sections here? In general, I'd prefer consistency unless the reason for the exception is apparent.

Grouping all of the `li` seems sensible but it's arguable that they shoudl group with the `ul`.